apps/web/src/components/ai/DataAgentChatAccessibility.tsx

```
@@ -0,0 +1,618 @@
+
+  const renderItem = (item: any, index: number) => {
+    const itemId = `${testId}-item-${index}`;
+    
+    switch (type) {
+      case 'clients':
+        return (
+          <div 
+            key={index} 
+            className="p-3 border rounded-md focus-within:ring-2 focus-within:ring-[#AC9469] focus-within:ring-offset-2"
+            role="article"
+            aria-labelledby={`${itemId}-name`}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<article>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)